runtime/src/lib.rs

```diff
@@ -297,13 +304,22 @@ impl pallet_template::Config for Runtime {
     type Event = Event;
 }
 
+parameter_types!{
+    pub const MinimumStake: u128 = 40_000 * 1_000 * 100_000_000;
+    pub const RewardReductionPeriod: BlockNumber = MINUTES * 5;
```

Comment:
Does this mean we reduce the reward every 5 blocks?

Reply:
yes